pkg/setting/setting.go

```diff
@@ -282,6 +282,8 @@ type Cfg struct {
 	AuthConfigUIAdminAccess bool
 	// TO REMOVE: Not documented & not supported. Remove with legacy handlers in 10.2
 	AuthBrokerEnabled bool
+	// TO REMOVE: Not documented & not supported. Remove in 10.3
+	TagAuthedDevices bool
```

Comment:
nit: is it tagged devices or is it the AuthedDevices that needs tagging.

the second is the one i am reading